dev/com.ibm.ws.concurrent/src/com/ibm/ws/concurrent/internal/ManagedScheduledExecutorServiceImpl.java

```diff
@@ -206,6 +206,13 @@ protected void setConcurrencyPolicy(ConcurrencyPolicy svc) {
         super.setConcurrencyPolicy(svc);
     }
 
+    @Override
+    @Reference(policy = ReferencePolicy.STATIC)
+    @Trivial
+    protected void setConcurrencyService(ConcurrencyService svc) {
```

Comment:
Nice, I'll look into switching to that option for this and some other pre-existing attributes that were following this pattern.  I'll make that improvement in a separate pull because this already has a successful build with all tests passing.